tools/lou_checkyaml.c

```diff
@@ -306,6 +306,10 @@ read_table(yaml_event_t *start_event, yaml_parser_t *parser, const char *display
 				"Table %s not valid", v->name);
 	free(emph_classes);
 	emph_classes = lou_getEmphClasses(v->name);	 // get declared emphasis classes
+	if (emph_classes == NULL) {
+		error_at_line(EXIT_FAILURE, 0, file_name, start_event->start_mark.line + 1,
+				"No declared emphasis classes found in table %s", v->name);
+	}
```

Comment:
I suggest that instead of terminating the program, we do a NULL check before using the variable below on line 655?

Reply:
Should I rebase the branch, edit the commit, or add a new commit to perform the modification?

Reply:
Clearest for us is a new commit, and then we can do the rebasing.

Reply:
@benehalo Never mind, it's not so important. We decided to just change the error message.